docs/conf.py

```diff
@@ -92,6 +95,7 @@ def resolve_xref(self, env, fromdocname, builder, typ, target, node, contnode):
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = []
 
+extlinks = {"github_nvflare_link": (f"https://github.com/NVIDIA/NVFlare/tree/{build_version}/%s", "")}
```

Comment:
NVFlare/tree or NVFlare/blob?

Reply:
Currently github redirects to either tree or blob based on if it is a directory or a file, so the link will work regardless. I put one version using tree for simplicity, but should we instead have two versions for blob/tree for clarity?

Reply:
Got it, thanks for the explanation, either way is good to me